### PR TITLE
Add table twilio_account_balance

### PR DESCRIPTION
**Pull request by reva**
# Example query results

```sql
select 
  *
from twilio_account_balance
```

<details>
  <summary>Results</summary>

```
-[ RECORD 1  ]---------------------------------------------------------------------------
balance     | 42.69
currency    | USD
account_sid | ACabcdefg1234567890
_ctx        | {"connection_name":"twilio"}
```
</details>

### CODE REVIEW

**Comment by misraved** [comment]
Welcome to Steampipe @reva and thanks a lot for raising a PR to add a new table 👍.

We recently released new version of the `twilio` plugin updating the Steampipe plugin SDK` to `v5.3.0` in https://github.com/turbot/steampipe-plugin-twilio/pull/20

Could you please update the PR by pulling the main into this branch?

**Comment by reva** [comment]
@misraved 
Thank's for having a look 🙂 I applied the changes and rebased on main. Can you check if it looks good now?

**Comment by reva** [comment]
@misraved 
Thank's for having a look 🙂 I applied the changes and rebased on main. Can you check if it looks good now?

**Comment by reva** [comment]
@misraved 
Thank's for having a look 🙂 I applied the changes and rebased on main. Can you check if it looks good now?

File: docs/tables/twilio_account_balance.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+## Examples
+
+### Fetch the balance
+
+```sql
+select
+    concat(balance, ' ', currency) as balance_formatted
+from
+    twilio_account_balance
```
Comment (misraved) [inline docs/tables/twilio_account_balance.md:13]:
```suggestion
select
    concat(balance, ' ', currency) as balance_formatted
from
    twilio_account_balance;
```

**Comment by misraved** [comment]
Hello @reva. as a part of the last leg of the review process, could you please run `gofmt -w twilio/table_twilio_account_balance.go` command to correct the formatting issues?

Thanks once again for this new table 👍.

**Comment by reva** [comment]
Sorry @misraved , first time with go here 😁 I ran `gofmt -w twilio/table_twilio_account_balance.go` but it didn't seem to change anything?

**Comment by misraved** [comment]
Thanks @reva for trying out the command, maybe GitHub showed a weird formatting issue 👍. 

Great work on the table 😄.

**Review by misraved — APPROVED** [review]

**Pull request merged**